lib/broadside/version.rb

```diff
@@ -1,3 +1,3 @@
 module Broadside
-  VERSION = '1.0.3'
+  VERSION = '1.1.0'
```

Comment:
i'll handle the version bump in a separate PR (need to update changelog and whatnot)